src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -509,21 +530,33 @@ public double rotationsToRadians(double rotations) {
 		return rotations * 2 * Math.PI;
 	}
 
+	public boolean isArmNearGoal() {
+		return Math.abs(getElbowPosition() - armPID.getGoal().position) <= 0.1;
```

Comment:
0.1 should be a named constant somewhere. And it likely should be a smaller number since 0.1 is about a third of the way from the bottom to the top (about 25-30 degrees). Note that changing this to something closer to what autonomous or a human driver expects will have unintended consequences.  See comment on `SetArmCommand` above

Reply:
There are also the (I believe unused) `ARM_POS_TOLERANCE ` and `WRIST_POS_TOLERANCE` constants - it would be good to use them here.